geoportailv3/static/less/geoportailv3.api.less

```diff
@@ -1 +1,77 @@
 @import (inline) '../../../node_modules/openlayers/css/ol.css';
+@import "../../../node_modules/bootstrap/less/mixins/vendor-prefixes.less";
+
+
+@popup-bg: #fff;
+@popup-border-color: rgba(0,0,0,0.2);
+@popup-arrow-outer-color: @popup-border-color;
+@popup-arrow-color: @popup-bg;
+@popup-arrow-outer-width: 11px;
+@popup-arrow-width: 11px;
```

Comment:
I reused the same rules/variables as bootstrap. We indeed don't need so much complexity.